src/client/java/teammates/client/scripts/DataMigrationForTimezoneFieldInCourses.java

```diff
@@ -0,0 +1,135 @@
+package teammates.client.scripts;
+
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import javax.jdo.JDOHelper;
+import javax.jdo.PersistenceManager;
+
+import teammates.client.remoteapi.RemoteApiClient;
+import teammates.common.datatransfer.CourseAttributes;
+import teammates.common.datatransfer.FeedbackSessionAttributes;
+import teammates.common.exception.EntityDoesNotExistException;
+import teammates.common.exception.InvalidParametersException;
+import teammates.common.util.Const;
+import teammates.logic.api.Logic;
+import teammates.storage.datastore.Datastore;
+import teammates.storage.entity.Course;
+
+public class DataMigrationForTimezoneFieldInCourses extends RemoteApiClient {
+    
+    public static Map<String, String> timeZoneDoubleToIdMapping = new HashMap<String, String>();
+    
+    static {
+        map("-12.0", "Etc/GMT+12");
+        map("-11.0", "US/Samoa");
+        map("-10.0", "US/Hawaii");
+        map("-9.5", "Pacific/Marquesas");
+        map("-9.0", "US/Alaska");
+        map("-8.0", "America/Los_Angeles");
+        map("-7.0", "America/Phoenix");
+        map("-6.0", "America/Chicago");
+        map("-5.0", "America/New_York");
+        map("-4.5", "America/Caracas");
+        map("-4.0", "America/Halifax");
+        map("-3.5", "America/St_Johns");
+        map("-3.0", "America/Sao_Paulo");
+        map("-2.0", "America/Noronha");
+        map("-1.0", "Atlantic/Cape_Verde");
+        map("0.0", "UTC");
+        map("1.0", "Europe/Paris");
+        map("2.0", "Europe/Athens");
+        map("3.0", "Africa/Nairobi");
+        map("3.5", "Asia/Tehran");
+        map("4.0", "Asia/Dubai");
+        map("4.5", "Asia/Kabul");
+        map("5.0", "Asia/Tashkent");
+        map("5.5", "Asia/Colombo");
+        map("5.75", "Asia/Kathmandu");
+        map("6.0", "Asia/Almaty");
+        map("6.5", "Asia/Rangoon");
+        map("7.0", "Asia/Jakarta");
+        map("8.0", "Asia/Singapore");
+        map("8.75", "Australia/Eucla");
+        map("9.0", "Asia/Tokyo");
+        map("9.5", "Australia/Adelaide");
+        map("10.0", "Australia/Canberra");
+        map("10.5", "Australia/Lord_Howe");
+        map("11.0", "Pacific/Noumea");
+        map("12.0", "Pacific/Auckland");
+        map("12.75", "Pacific/Chatham");
+        map("13.0", "Pacific/Tongatapu");
+        map("14.0", "Pacific/Kiritimati");
```

Comment:
All these values are arbitrarily chosen, although there are obvious biases such as `Asia/Jakarta` and `Asia/Singapore` :p